benchmarks/Microsoft.AspNetCore.Routing.Performance/Matchers/MatcherAzureBenchmark.cs

```diff
@@ -1,20 +1,18 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
 using System.Threading.Tasks;
 using BenchmarkDotNet.Attributes;
 
 namespace Microsoft.AspNetCore.Routing.Matchers
 {
     // Generated from https://github.com/Azure/azure-rest-api-specs
-    public partial class AzureMatcherBenchmark : MatcherBenchmarkBase
+    public partial class MatcherAzureBenchmark : MatcherBenchmarkBase
     {
         private const int SampleCount = 100;
 
         private BarebonesMatcher _baseline;
         private Matcher _dfa;
-        private Matcher _route;
```

Comment:
I'm taking the legacy route implementation out of these big tests, because it's just so slow. It makes the benchmarks really painful to work with and isn't adding much value because it's like 100x slower than the baseline.